services/processor/native/sanitize_code.go

```diff
@@ -24,39 +24,40 @@ func (s *service) createSanitizer() *sanitizer.Sanitizer {
 
 func SanitizerConfigForProduction() *sanitizer.SanitizerConfig {
 	return &sanitizer.SanitizerConfig{
-		ImportWhitelist: map[string]bool{
+		ImportWhitelist: map[string]string{
+			// package: reason to whitelist
+
 			// SDK
-			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1"`:          true,
-			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/address"`:  true,
-			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/env"`:      true,
-			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/ethereum"`: true,
-			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/events"`:   true,
-			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/safemath"`: true,
-			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/service"`:  true,
-			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/state"`:    true,
+			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1"`:          "SDK",
```

Comment:
this code runs upon deployment, right? so SLA isn't SUPER critical?
if so, I'd consider not using a map, but rather creating a collection of whitelisted libs, with support for glob patterns, so that we can do `github.com/orbs-network/orbs-contract-sdk/go/sdk/*`

it's ok to leave it like this now and open an issue, but keep in mind that right now, on every package added to the sdk we'll need to add it here.